galaxychop/galaxychop.py

```diff
@@ -92,54 +92,127 @@ class Galaxy:
+    coordinates x_s, y_s, z_s of the stars and rotates them to 
+    align them in the direction of J. 
+    
+    vel_star_rot = np.array(n, 3), takes velocities vx_s, vy_s, vz_s 
+    of the stars and rotates them to align them in the direction of J.
+    
+    A = np.array(3, 3), rotation matrix.
+    
+    pos_dark_rot = np.array(m, 3), takes coordinates x_dm, y_dm, z_dm of the 
+    dark matter and rotates them repect the A matrix.
+    
+    vel_dark_rot = np.array(m, 3), takes velocities vx_dm, vy_dm, vz_dm of the
+    dark matter and rotates them repect the A matrix.
+    
+    pos_gas_rot = np.array(g, 3), takes coordinates x_g, y_g, z_g of the 
+    gas and rotates them repect the A matrix.
+    
+    vel_gas_rot = np.array(g, 3), takes velocities vx_g, vy_g, vz_g of the 
+    gas and rotates them repect the A matrix.
+    """
+
+    pos_star_rot, vel_star_rot, A = aling(m_s, [x_s, y_s, z_s], 
+                                        [vx_s, vy_s, vz_s], 3 .R[j])
+
+    pos_dark_rot = rot([x_dm, y_dm, z_dm], A)
+    vel_dark_rot = rot([vx_dm, vy_dm, vz_dm], A)
+
+    pos_gas_rot = rot([x_g, y_g, z_g], A)
+    vel_gas_rot = rot([vx_g, vy_g, vz_g], A)
+
+    """Calculation of the components of the angular momentum 
+    of stars, gas and DM.
+   
+    Parameters
+    ----------
+   
+    pos_star_rot, vel_star_rot. 
+    pos_dark_rot, vel_dark_rot. 
+    pos_gas_rot, vel_gas_rot.
+   
+    Returns
+    -------
+   
+   L_part = np.array(3, 3*n), concatenates angular momentum for the stars, 
+   dark matter and gas in a single array.
+   
+   Lr_star = np.array(n, 1), Component in the plane of the angular 
+   momentum of stars. 
+   
+   Lr = np.array(3*n, 1) Component in the plane of the angular 
+   momentum for all particles.
+   
+    """
+
+    L_dark = np.asarray((pos_dark_rot[:, 1] * vel_dark_rot[:, 2] -
+                         pos_dark_rot[:, 2] * vel_dark_rot[:, 1],
+                         pos_dark_rot[:, 2] * vel_dark_rot[:, 0] -
+                         pos_dark_rot[:, 0] * vel_dark_rot[:, 2],
+                         pos_dark_rot[:, 0] * vel_dark_rot[:, 1] -
+                         pos_dark_rot[:, 1] * vel_dark_rot[:, 0]))
+
+    L_star = np.asarray((pos_star_rot[:, 1] * vel_star_rot[:, 2] -
+                         pos_star_rot[:, 2] * vel_star_rot[:, 1],
+                         pos_star_rot[:, 2] * vel_star_rot[:, 0] -
+                         pos_star_rot[:, 0] * vel_star_rot[:, 2],
+                         pos_star_rot[:, 0] * vel_star_rot[:, 1] -
+                         pos_star_rot[:, 1] * vel_star_rot[:, 0]))
+
+    L_gas = np.asarray((pos_gas_rot[:, 1] * vel_gas_rot[:, 2] - 
+                        pos_gas_rot[:, 2] * vel_gas_rot[:, 1],
+                        pos_gas_rot[:, 2] * vel_gas_rot[:, 0] - 
+                        pos_gas_rot[:, 0] * vel_gas_rot[:, 2],
+                        pos_gas_rot[:, 0] * vel_gas_rot[:, 1] - 
+                        pos_gas_rot[:, 1] * vel_gas_rot[:, 0]))
```

Comment:
I don't think this corresponds to reading the data from the simulation. It looks more like the rotation of the coordinate system to align it with the angular momentum. Don't worry, I'll open the issue that corresponds to what you did so that after you modify it you can link it.

Keep in mind that this should be a method within the galaxy class.